assets/app/view/game/corporation.rb

```diff
@@ -147,14 +147,23 @@ def render_title
           style: {
             color: 'currentColor',
             display: 'inline-block',
-            justifySelf: 'start',
+            justifySelf: 'center',
           },
         }
+        children = [h(:img, logo_props),
+                    h('div.title', name_props, @corporation.full_name)]
 
-        h('div.corp__title', title_row_props, [
-          h(:img, logo_props),
-          h('div.title', name_props, @corporation.full_name),
-        ])
+        if @corporation.system?
+          logo_props[:attrs][:src] = logo_for_user(@corporation.corporations.last)
+        else
+          logo_props[:attrs][:src] = 'data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7'
```

Comment:
what is this????

Reply:
Seems to be an empty placeholder. Bad style @crericha 
The element should only be there `if @corporation.system?` so corp title gets centered either way. Test against 1889.

Reply:
Yes, an empty image. It was the recommended solution on stackoverflow. I wasn't too sure about it myself and one of the reasons I wanted yzemaze's input here. My CSS skills are at the beginner level.  `grid: '1fr / repeat(auto-fit, auto)'` solved my column problem.

I cloned an 1889 game and verified the corporation cards look the same on 18xx.games and my local version.